src/main/java/pro/sky/skyprospringcoursework21/ExaminerServiceImpl.java

```diff
@@ -11,7 +11,7 @@
 public class ExaminerServiceImpl implements ExaminerService{
     private final QuestionService questionService;
 
-//    private  final List<Question> mapExaminerService = new ArrayList<>();
+    private  final HashSet<Question> mapExaminerService = new HashSet<>();
```

Comment:
```suggestion
    private  final Set<Question> mapExaminerService = new HashSet<>();
```
тип ссылки всегда должен быть более общий, чем тип объекта

Reply:
здесь не должно быть никакого поля, зачем хранить вопросы в сервисе? это должна быть локальная переменная метода getQuestions